### DISCUSSION TITLE
Docs: So is "Signed-off-by" in commit messages required, or not?

### DISCUSSION THREAD

**Discussion by pbodnar** [eclipse-platform/.github#30] [Q&A] [answered]
Hi, I'm referring to [this](https://github.com/eclipse-platform/.github/blame/main/CONTRIBUTING.md#L105) part of CONTRIBUTING.md:

> Commits that are provided by non-committers must have a Signed-off-by field in the footer ...

This seems to go against what was done in [Bug 558653](https://bugs.eclipse.org/bugs/show_bug.cgi?id=558653) - i. e. this requirement was _removed_ from all the various documentation some year ago.

So is this just a typo in this new CONTRIBUTING.md, or did I miss something?

**Comment by merks** [comment, marked as answer]
The Signed-off-by is not required anymore.

**Reply by mickaelistria** [reply]
+1 with that. @pbodnar as you identified the issue, would you mind submitting a PR updating the CONTRIBUTING.md ? So it will be forever visible in history that this guide is better thanks to you ;)

**Reply by pbodnar** [reply]
Thank you for your replies, so here we go: #32. I have somewhat widened the documentation changes while at it. ;)
